functests/1_performance/performance.go

```diff
@@ -671,6 +671,15 @@ func verifyV2Conversion(v2Profile *performancev2.PerformanceProfile, v1Profile *
 		}
 	}
 
+	for _, f := range v2Profile.GetObjectMeta().GetManagedFields() {
+		if f.APIVersion == performancev1alpha1.GroupVersion.String() ||
+			f.APIVersion == performancev1.GroupVersion.String() {
+			if v2Profile.Spec.GloballyDisableIrqLoadBalancing == nil || !*v2Profile.Spec.GloballyDisableIrqLoadBalancing {
+				return fmt.Errorf("globallyDisableIrqLoadBalancing field must be set to true")
```

Comment:
must be set as in must have a value ?  can't it be nil ?  its defined as omitempty : 
`json:"globallyDisableIrqLoadBalancing,omitempty"`

Reply:
Yes, is on purpose, when converting a v1 profile to v2 we must set the flag to true on order to keep the irq load balancing disabled as in v1 profile.